rexrov2_description/urdf/rexrov2_base.xacro

```diff
@@ -15,10 +15,10 @@
      limitations under the License.
 -->
 <robot xmlns:xacro="http://www.ros.org/wiki/xacro" >
-  <xacro:include filename="$(find uuv_descriptions)/urdf/common.urdf.xacro"/>
-  <xacro:include filename="$(find uuv_sensor_ros_plugins)/urdf/sensor_snippets.xacro"/>
+  <xacro:include filename="$(find rexrov2_description)/urdf/common.urdf.xacro"/>
+  <!-- <xacro:include filename="$(find uuv_sensor_ros_plugins)/xacro/sensor_snippets.xacro"/> -->
```

Comment:
it is nice to add a comment to why you commented the code so can track in the future